src/app/test/test-folder-tree/test-folder-tree.component.ts

```diff
@@ -20,7 +20,7 @@ export class TestFolderTreeComponent {
     folderBehaviourOnClick: 'select',
     expandAllFolders: true,
     highlightOpenFolders: false,
-    autoSelectCondition: (item: CreateTreeItem) => this.autoSelectItem(item),
+    autoOpenCondition: (item: CreateTreeItem) => this.autoSelectItem(item),
```

Comment:
Why did you change this?

Reply:
It told me that it was wrong and it caused a compile error

Reply:
with "wrong" i mean it said it didnt exist on FileTreeOptions

Reply:
Did you run `yarn install`?

Reply:
i did not, i will do so immediately

Reply:
i ran `yarn install` and changed it back